specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -260,6 +260,242 @@
+          "BillingAccountAlerts": {
+            "$ref": "./examples/BillingAccountAlerts.json"
+          },
+          "BillingProfileAlerts": {
+            "$ref": "./examples/BillingProfileAlerts.json"
+          },
+          "InvoiceSectionAlerts": {
+            "$ref": "./examples/InvoiceSectionAlerts.json"
+          },
+          "EnrollmentAccountAlerts": {
+            "$ref": "./examples/EnrollmentAccountAlerts.json"
+          },
+          "DepartmentAlerts": {
+            "$ref": "./examples/DepartmentAlerts.json"
+          },
+          "SubscriptionAlerts": {
+            "$ref": "./examples/SubscriptionAlerts.json"
+          },
+          "ResourceGroupAlerts": {
+            "$ref": "./examples/ResourceGroupAlerts.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/scopeAlertParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/AlertsResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.CostManagement/{externalCloudProviderType}/{externalCloudProviderId}/alerts": {
+      "get": {
+        "tags": [
+          "Alerts"
+        ],
+        "operationId": "Alerts_ListExternal",
+        "description": "Lists the Alerts for external cloud provider type defined.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "ExternalBillingAccountAlerts": {
+            "$ref": "./examples/ExternalBillingAccountAlerts.json"
+          },
+          "ExternalSubscriptionAlerts": {
+            "$ref": "./examples/ExternalSubscriptionAlerts.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/externalCloudProviderTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/externalCloudProviderIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/AlertsResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/forecast": {
+      "post": {
```

Comment:
>post [](start = 7, length = 4)

Why is the list operation implemented using POST? Is it because you're supporting OData filters?

Reply:
it has a request body which a GET wouldn't support. Forecast calls Query and Query uses POST